cypress/e2e/testing/login.cy.js

```diff
@@ -1,5 +1,5 @@
-describe("Logging", () => {
-  it("the user can log in", () => {
+describe("Log in", () => {
+  it("log in and landing on the profile page", () => {
     cy.visit("https://workflowkj.netlify.app/");
 
     cy.scrollTo(0, 500);
```

Comment:
Interesting that you need to scroll down before the test can interact with the button. This can be due to animations. I way to deal with this through the entire project by disabling all animations/ transitions in test but that still defeats the purpose of e2e testing. It's a tricky one.